### PR TITLE
Fix/dedicated layout with bootstrap

### PR DESCRIPTION
**Pull request by Heiiji**
| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| License          | BSD 3-Clause

## Description

update 'offset' class on bootstrap 4 references to fox responssive result.

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/index.html [hunk 1]
```diff
@@ -75,9 +75,7 @@
                         data-sidebar-namespace="'account'"
                     ></div>
 
-                    <div
-                        class="col-md-9 col-md-offset-3 col-lg-10 col-lg-offset-2 px-0"
-                    >
+                    <div class="col-lg-10 offset-lg-2 px-0">
```
Comment (cbourgois) [inline packages/manager/apps/dedicated/client/app/index.html:78]: `md` not required ?

Comment (Heiiji) [reply packages/manager/apps/dedicated/client/app/index.html:78]: Nope, other CSS rule discard side-menu before `md` rule. :)

Comment (marie-j) [reply packages/manager/apps/dedicated/client/app/index.html:78]: I'm quite confused with this change as the same component and same configuration is applied on other universe (e.g: web https://github.com/ovh/manager/blob/master/packages/manager/apps/web/client/app/index.html#L68) and it doesn't seem to cause any issue or if it does maybe it should be applied on other universes

Comment (Heiiji) [reply packages/manager/apps/dedicated/client/app/index.html:78]: for apply on other universe, we need to import same version of bootstrap everywhere :/

**Pull request closed without merging**